.eslintrc.js

```diff
@@ -398,7 +398,7 @@ module.exports = {
         'import/no-extraneous-dependencies': [
           'error',
           {
-            packageDir: resolve(__dirname, 'x-pack/package.json'),
```

Comment:
__question:__ do you really need to do all these changes in this file? The less changes we make, the better (if possible).